src/main/drivers/timer.h

```diff
@@ -57,6 +57,11 @@ typedef uint32_t timCCR_t;
 typedef uint32_t timCCER_t;
 typedef uint32_t timSR_t;
 typedef uint32_t timCNT_t;
+#elif defined(STM32G4)
```

Comment:
Should we actually contract these? There seem to be only two variants of the `typedef`s.